mlflow/models/evaluation/base.py

```diff
@@ -1359,6 +1379,8 @@ def evaluate(
           metrics.
         - **col_mapping**: A dictionary mapping column names in the input dataset or output
           predictions to column names used when invoking the evaluation functions.
+        - **k**: The number of top retrieved documents to use when computing the built-in metric
+          precision_at_k for model type "retriever". Default value is 3.
```

Comment:
nit, can we add something like "For other model types, this parameter will be ignored."